tests/medic-conf/medic-conf-actions.spec.js

```diff
@@ -0,0 +1,32 @@
+const {expect} = require('chai');
+const util = require('util');
+const exec = util.promisify(require('child_process').exec);
+const url = require('../utils').getInstanceUrl();
+
+//Not all actions tested here due to missing forms and config
+//[convert-collect-forms , upload-collect-form, upload-branding, upload-partners, upload-privacy-policies]
+const actions = ['compile-app-settings','backup-app-settings','convert-app-forms','convert-contact-forms',
+  'backup-all-forms','upload-app-forms','upload-contact-forms', 'upload-resources','upload-custom-translations'];
+
+describe('Medic-conf actions tests', () => {
+  const runCommand = async (action) => {
+    try {
+      const { stdout } = await exec(`medic-conf --url=${url} ${action}s --force`, { cwd: 'config/default' });
```

Comment:
@ngaruko your tests are now failing because of the additional S you here (probably for testing that errors from medic-conf calls are displayed in logs).